src/playermat/Playmat.ttslua

```diff
@@ -457,133 +434,102 @@ function replenishTokens(card, count, replenish)
 
   local newCount = foundTokens + replenish
   if newCount > count then newCount = count end
-  spawnTokenGroup(card, "resource", newCount)
-end
-
-function getPlayerCardData(object)
-  return PLAYER_CARDS[object.getName()..':'..object.getDescription()] or PLAYER_CARDS[object.getName()]
+  tokenManager.spawnTokenGroup(card, "resource", newCount)
```

Comment:
Looks like this is part of the replenish function? Does not look intended to me since it doesn't take the replenish count as argument

Reply:
The diff on this is misleading.  It's replacing this line:
https://github.com/argonui/SCED/blob/ff7ed57bca58989372eb003193624f059f63b0e2/src/playermat/Playmat.ttslua#L460

It's identical except for using the TokenManager to do the spawning instead of the local method.